Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -605,6 +605,11 @@ def audit_desc
   def audit_homepage
     homepage = formula.homepage
 
+    if homepage.nil? || homepage.empty?
```

Comment:
Another option I tend to use in this case is `to_s.empty?` but I'm happy either way.